Packs/GitHub/Integrations/GitHub/GitHub.yml

```diff
@@ -63,6 +64,12 @@ configuration:
   name: installation_id
   required: false
   type: 0
+- display: API Token
+  name: token
+  required: false
+  type: 4
+  hidden: true
+  additionalinfo: use api_token instead
```

Comment:
WHy we added this?

Reply:
Because I need to avoid breaking bc so i'm keeping the old key for those who already have a pre-configured instance

Reply:
But the parameter is hidden so the users will not see it anyway